lib/samly/helper.ex

```diff
@@ -110,10 +110,24 @@ defmodule Samly.Helper do
 
   defp decode_saml_payload(saml_encoding, saml_payload) do
     try do
-      xml = :esaml_binding.decode_response(saml_encoding, saml_payload)
+      xml = decode_response(saml_encoding, saml_payload)
       {:ok, xml}
     rescue
       error -> {:error, {:invalid_response, "#{inspect(error)}"}}
     end
   end
+
+  defp decode_response(_, saml_payload) do
```

Comment:
should be direct copy of:
https://github.com/handnot2/esaml/blob/main/src/esaml_binding.erl#L39
+ comments option (https://www.erlang.org/doc/man/xmerl_scan.html#type-option_list)